code/src/Components/SendMessage.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/no-unescaped-entities */
+import React from 'react';
+
+const SendMessage = ({ newMessage, handleNewMessageChange, onFormSubmit }) => {
+  return (
+    <section className="outer-send-message-container">
+      <div className="inner-send-message-container">
+        <form onSubmit={onFormSubmit}>
+          <h2>What's making you happy right now?</h2>
+          <textarea
+            value={newMessage}
+            type="text"
+            placeholder="My happy thought is..."
+            onChange={handleNewMessageChange}
+            className="text-area"
+            maxLength="140"
+            minLength="10" />
```

Comment:
These ones are restricted otherplaces right? Because you can post messages that are 5 characters long, I tried it due to the button-disable at line 22.